plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/AbstractPinotIntegrationSmokeTest.java

```diff
@@ -296,6 +297,69 @@ protected QueryRunner createQueryRunner()
         pinot.createSchema(getClass().getClassLoader().getResourceAsStream("quotes_in_column_name_schema.json"), QUOTES_IN_COLUMN_NAME_TABLE);
         pinot.addRealTimeTable(getClass().getClassLoader().getResourceAsStream("quotes_in_column_name_realtimeSpec.json"), QUOTES_IN_COLUMN_NAME_TABLE);
 
+        // Create a table having multiple columns with the same values
```

Comment:
nit: "same values" -> "some duplicate values" might be more exact.

Reply:
yep, thanks:)